trieste/acquisition/rule.py

```diff
@@ -267,8 +267,15 @@ def acquire(
         if summary_writer:
             with summary_writer.as_default(step=step_number):
                 batched_points = tf.expand_dims(points, axis=0)
-                value = self._acquisition_function(batched_points)[0][0]
-                logging.scalar("EGO.acquisition_function/maximum_found" + "[0]" * greedy, value)
+                values = self._acquisition_function(batched_points)[0]
+                if len(values) == 1:
+                    logging.scalar(
+                        "EGO.acquisition_function/maximum_found" + "[0]" * greedy, values[0]
+                    )
+                else:  # vectorized acquisition function
+                    logging.histogram(
```

Comment:
do we want to do something more complicated than than, e.g. also track individual values? or is this overkill?

Reply:
what do you mean? generate individual scalar plots for the different batch elements, like we do for greedy batch?

Reply:
yes

Reply:
I'm not entirely sure it's a good idea considering that the batch isn't following any particular order so we might see lots of random jumps

Reply:
yeah. with greedy at least there is an explicit ordering.